README.md

```diff
@@ -171,6 +171,19 @@ interval between any two periodic sync events can be greater than the requested
 is up to the browser implemetation. Also, the interval between any two periodic sync events for a
 registration doesn't have to be the same.
```

Comment:
Should we amend the above paragraph to align better with what follows? (Or vice versa.) Particularly, here we discuss _intervals_, in the new text we discuss _frequencies_.

Reply:
Both are discussing time, that doesn't have to be confusing, does it?
I think talking about intervals in this paragraph when discussing minInterval is more natural. I'll try and update the next paragraph to talk about intervals instead of frequencies.